src/testrel.jl

```diff
@@ -588,7 +588,11 @@ function _test_rel_step(
             end
 
             if !step.expect_abort
-                @test state == "COMPLETED" && !unexpected_errors_found
+                @test state == "COMPLETED"
+                @test !unexpected_errors_found
```

Comment:
could we also log the unexpected errors? maybe change `unexpected_errors_found` to a vector instead of a bool and check that its length is zero? I think that will print out what is there if it is incorrect (maybe?)

Reply:
They're logged up on line 576.